pkg/sql/crdb_internal.go

```diff
@@ -197,7 +197,36 @@ CREATE TABLE crdb_internal.tables (
   drop_time                TIMESTAMP,
   audit_mode               STRING NOT NULL
 )`,
-	populate: func(ctx context.Context, p *planner, _ *DatabaseDescriptor, addRow func(...tree.Datum) error) error {
+	populate: func(ctx context.Context, p *planner, dbContext *DatabaseDescriptor, addRow func(...tree.Datum) error) error {
+		// Virtual descriptors first.
+		vt := p.getVirtualTabler()
+		vEntries := vt.getEntries()
+		vSchemaNames := vt.getSchemaNames()
+		for _, virtSchemaName := range vSchemaNames {
+			e := vEntries[virtSchemaName]
+			for _, tName := range e.orderedDefNames {
+				table := e.defs[tName].desc
+				if err := addRow(
```

Comment:
make a single function that takes the desc as argument and does the common thing between this and below